parsely-parsely-page.php

```diff
@@ -4,6 +4,9 @@
    <script type="application/ld+json">
    <?php echo json_encode($parselyPage); ?>
    </script>
+    <?php if (isset($parselyPage["custom_metadata"])) : ?>
+        <meta name='parsely-metadata' content='<?php echo json_encode($parselyPage['custom_metadata'])?>'>
+        <?php endif; ?>
```

Comment:
Where are we exposing the option for them to set the custom metadata and to input the custom values?